.bazelrc

```diff
@@ -3,6 +3,12 @@ try-import submodules/dev-tools/.bazelrc
 # Specific Bazel build/test options.
 build --enable_platform_specific_config
 
+build:macos --cxxopt='-std=c++17'
+
+build:linux --cxxopt='-std=c++17'
+
+build:windows --cxxopt="/std:c++17"
```

Comment:
These should be specified in [bazel/copts](https://github.com/3rdparty/eventuals/blob/main/bazel/copts.bzl). Can you update this to add a comment that points people there so folks don't try and do this in the future?